src/mentionui.js

```diff
@@ -374,7 +374,9 @@ export default class MentionUI extends Plugin {
 			this._balloon.add( {
 				view: this._mentionsView,
 				position: this._getBalloonPanelPositionData( markerMarker, this._mentionsView.position ),
-				withArrow: false
+				withArrow: false,
+				singleViewMode: true,
+				stack: 'mention'
```

Comment:
Why is this needed?

Reply:
The `stack` or whole `singleViewMode`? Without the `stack` the mention panel would be added to the `'main'` stack. As the mention ui should be displayed in single view mode I recon that adding it to a separate stack would add separation from other features.

Reply:
What kind of separation? Doesn't `singleViewMode` guarantee complete separation already?

Reply:
Separation of stacks: Without `stack: 'mention'` it still would be displayed as a single view but internally it will reside in a stack with other features. 

So adding a new stack for single view is not obligatory I feel that it is much cleaner to have own stack for SVM rather then using `'main'` stack.

Reply:
I also do not know why you need to define a separate stack here. Also, the property is `stackId` not `stack` and the fact that it works fine with the wrong property proved that it is not needed ;)

Reply:
:man_facepalming: on me then

Reply:
I've got a good smell, then ;) Please remove it on master.

Reply:
done